README.md

```diff
@@ -106,6 +106,8 @@ Reference the Maven extension in the `.mvn/extensions.xml` file as described abo
 *Prerequisites:*
 
 * Add the `Cloud Manager API` to your Adobe IO project at [https://console.adobe.io/](https://console.adobe.io/)
+  * Add a Service Account (JWT based) by either upload private/public keypairs or letting the wizard generate those for you
```

Comment:
let me clarify further, that you only need to upload the public key in case you create on your own and that you have to download the private key immediately after generating.